projects/epbs-with-inclusion-list.md

```diff
@@ -0,0 +1,90 @@
+2. Proposers of current slot can send a list of transactions which they want to get included by the next block's builder to avoid censorship. This will be tackled in a different way to avoid free DA problem as mentioned in the post above.
+
+## Roadmap
+
+*What is your proposed timeline? Outline parts of the project and insight on how much time it will take to execute them.*
+
+**Week 1-3** Spend time on learning more about PBS and ePBS desigs. Read and review the specs thoroughly (and if possible contiribute to it) and figure out the implementation plans. Take part in discussions about the design choices with mentors and other fellows.
+
+**Week 4-10**: Divide the modules to be implemented and start with iterative development on CL and EL. Plan the implementation in a way where each module (or a group of them) can be tested (via unit tests for now) in isolation. Based on the progress, plan the deployment of the PoC on a devnet.
+
+**Week 11-12**: Document the project and continue with the testing plans.
+
+## Possible challenges
+
+*What are the limitations and issues you may need to overcome?*
+1. ePBS is still being actively researched upon and as mentioned above there are multiple designs available. The community haven't finalised as of now, which design will be suitable more and hence there's a possibility that it might change in the future.
+2. It's a huge project and even with multiple people working on it it might take more time then expected to complete it. If not testing, we hope to complete the PoC implementation by the end of the fellowship. I would be happy to continue working on it after the fellowship as well.
+3. At this point, I see that there are lot of implementation unknows and a lot are expected as we proceed (at least according to me). It will challenging to tackle them and also work with a huge codebase of Prysm and Geth.
+
+## Goal of the project
+
+*What does success look like? Describe the end goal of the project, scope, state and impact for the project to be considered finished and successful.*
+
+The end goal of the project is to review the existing literature, specs and have a working implementation of ePBS PTC design with inclusion list on a devnet. Here is the rough list of things which we expect to complete (as a group):
+
+1. PoC implementation in Prysm (CL) and Geth (EL).
+2. Testing plan for the PoC (included unit tests as well as devnet testing)
+3. Documentation of the project
+
+## Collaborators
+
+### Fellows 
+
+*Are there any fellows working with you on this project?*
+
+As it's a huge project, yes there are other fellows also working on this with me. 
+- [Anshal Shukla](https://github.com/anshalshukla)
+- [Chirag Mehta](https://github.com/Chirag018)
+- [NC](https://github.com/naviechan) (An external participant)
+- [Ella](https://github.com/0xfmoi) (Haven't discussed with her yet, but saw her proposal on the same topic)
```

Comment:
I see her proposal overlaps with yours and if you are going to collaborate, I would prefer merging them in favor of yours.